scio-bigquery/src/main/scala/com/spotify/scio/bigquery/taps.scala

```diff
@@ -133,8 +132,8 @@ final case class BigQueryTaps(self: Taps) {
       () =>
         BigQueryStorage(Table.Ref(table)).tap(
           BigQueryStorage.ReadParam(
-            readOptions.getSelectedFieldsList().asScala.toList,
-            readOptions.getRowRestriction()
+            readOptions.getSelectedFieldsList.asScala.toList,
+            Some(readOptions.getRowRestriction)
```

Comment:
`Option` :)

Reply:
this one is actually ok coz getRowRestriction returns empty String if not set

Reply:
changed it coz on a second thought .. i don't wanna rely on third party behavior when i can play safe